src/Layers/xrRender/D3DXRenderBase.h

```diff
@@ -56,7 +56,7 @@ class D3DXRenderBase : public IRender, public pureFrame
     xr_vector<R_dsgraph::mapNormalTextures::value_type *> nrmTexturesTemp;
 
     xr_vector<R_dsgraph::mapMatrixVS::value_type *> matVS;
-#ifndef USE_DX9
+#if defined(USE_DX11) || defined(USE_OGL)
```

Comment:
Many modern graphics APIs have geometry shaders, so, let's better to leave it as is:
```suggestion
#ifndef USE_DX9
```